base/linalg/bidiag.jl

```diff
@@ -40,6 +40,15 @@ function getindex{T}(A::Bidiagonal{T}, i::Integer, j::Integer)
     i == j ? A.dv[i] : (A.isupper && (i == j-1)) || (!A.isupper && (i == j+1)) ? A.ev[min(i,j)] : zero(T)
 end
 
+## structured matrix methods ##
+function Base.replace_in_print_matrix(A::Bidiagonal,i::Integer,j::Integer,s::AbstractString)
+  if A.isupper
```

Comment:
I just made the changes now in nano (I'm in the boondocks on another machine), but I wrote everything before in atom.